### PR TITLE
added hasRepository method to RepositoryManagerInterface

### PR DESCRIPTION
**Pull request by imper86**
My use case:
I started developing custom data provider for ApiPlatform, which uses FOSElasticaBundle. With hasRepository method it will be easier to check if api resource is indexed with this bundle.

### CODE REVIEW

File: src/Doctrine/RepositoryManager.php [hunk 1]
```diff
@@ -79,4 +79,9 @@ public function getRepository($entityName)
 
         return $this->repositoryManager->getRepository($realEntityName);
     }
+
+    public function hasRepository($typeName): bool
```
Comment (XWB) [inline src/Doctrine/RepositoryManager.php]: You may type hint the parameter.

File: src/Manager/RepositoryManager.php [hunk 2]
```diff
@@ -73,6 +73,17 @@ public function getRepository($typeName)
         return $repository;
     }
 
+    /**
+     * Checks if repository exists for type
+     *
+     * @param $typeName
+     * @return bool
+     */
+    public function hasRepository($typeName): bool
```
Comment (XWB) [inline src/Manager/RepositoryManager.php]: You may type hint the parameter.

File: src/Manager/RepositoryManagerInterface.php [hunk 3]
```diff
@@ -43,4 +43,12 @@ public function addType($indexTypeName, FinderInterface $finder, $repositoryName
...
      */
     public function getRepository($typeName);
+
+    /**
+     * Checks if repository exists for type
+     *
+     * @param $typeName
+     * @return bool
+     */
+    public function hasRepository($typeName): bool;
```
Comment (XWB) [inline src/Manager/RepositoryManagerInterface.php]: You may type hint the parameter.

**Comment by COil** [comment]
Hi @imper86 , can you make the required changes please?

**Comment by XWB** [comment]
Ping @imper86

**Comment by imper86** [comment]
Sorry guys for not responding. I will close this PR because I started some temporary solution to make this bundle compatible with doctrine bundle v2 and, my mistake, I did it on the same branch as this PR.

I will open a new one in a few minutes.

**Pull request closed without merging**